wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -545,11 +552,21 @@ public static void updateBeanValue(final WComponent component)
             // Update the rest of the bean
             for (int i = ((Container) component).getChildCount() - 1; i >= 0; i--)
             {
-                updateBeanValue(((Container) component).getChildAt(i));
+                updateBeanValue(((Container) component).getChildAt(i), visibleOnly);
             }
         }
     }
-
+    
+        /**
+     * Updates the bean value with the current value of the component
+     * and all its bean-bound children. By default this method will process invisible containers and inputs.
+     *
+     * @param component the component whose contents need to be copied to the bean.
+     */
+    public static void updateBeanValue(final WComponent component)
+    {
+            updateBeanValue(component, false);

```

Comment:
Based on the discussions we had I decided to leave it false to ensure no impact to existing applications.

Personally I'd prefer it to process visible components only, but I understand the need to reduce upgrade impacts.